src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -64,8 +65,8 @@ const transform = entity => ({
           value: entity.fieldDatetimeRangeTimezone[0].value
             ? Date.parse(entity.fieldDatetimeRangeTimezone[0].value) / 1000
             : null,
-          endValue: entity.fieldDatetimeRangeTimezone[0].endValue
-            ? Date.parse(entity.fieldDatetimeRangeTimezone[0].endValue) / 1000
+          endValue: entity.fieldDatetimeRangeTimezone[0].end_value
+            ? Date.parse(entity.fieldDatetimeRangeTimezone[0].end_value) / 1000
```

Comment:
Any idea why `entity.fieldDatetimeRangeTimezone[0].end_value` is still snake case?

Reply:
The input is snake, output is camel.